mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
Do we need all the extra text to basically describe the four possible states of given domain in sending MTA STS cache: ENFORCING, PENDING_ENFORCING, REPORTING and MISSING.